packages/dataviews/src/style.scss

```diff
@@ -355,6 +356,31 @@
 			}
 		}
 	}
+
+	.dataviews-view-grid__badge-fields {
+		&:not(:empty) {
+			padding: $grid-unit-15;
+			padding-top: 0;
+		}
+
+		.dataviews-view-grid__field-value {
+			&.is-badge {
```

Comment:
Can we have a field within the `dataviews-view-grid__badge-fields` container that is not a badge field?

Reply:
That should be a leftover after my small refactoring. I'll update.